App/Services/Notifications.ts

```diff
@@ -9,23 +9,65 @@ export interface INotificationsPayload {
   typeString: string,
 }
 
-export function isPhoto(notification: Notification): boolean {
-  if (!notification.data_id) {
-    return false
-  }
-  switch (notification.type) {
-    case NotificationType.receivedInviteNotification:
+export function toTypedNotification(notificationData: NotificationData): Notification {
```

Comment:
This is the function that converts the untyped `NotificationData` to type safe notifications.